index.html

```diff
@@ -1106,8 +1104,8 @@ <h4>
       a certain amount of time, then you time out.
     </p>
     <pre class="example">
-      const reader = new NDEFReader();
-      reader.onreading(event => console.log("We read a tag!"));
+      const ndef = navigator.nfc.ndef;
```

Comment:
May we try without the shortcut `const ndef` for consistency and use `navigator.nfc.ndef` everywhere?